packages/calypso-e2e/src/lib/pages/editor-page.ts

```diff
@@ -39,7 +39,7 @@ const selectors = {
 	// Welcome tour
 	welcomeTourCloseButton: 'button[aria-label="Close Tour"]',
 };
-const EXTENDED_TIMEOUT = 90 * 1000;
+const EXTENDED_TIMEOUT = 30 * 1000;
```

Comment:
Not strictly related to this change. However, I feel reasonably safe that we can reduce the overly generous 90s timeout.